### PR TITLE
fix markdown on yum_cron_update_cmd

### PR DESCRIPTION
**Pull request by kpankonen**
there were a couple spots where the indented code blocks weren't being rendered correctly by github (the ones after a list)

### CODE REVIEW

**Comment by giovtorres** [comment]
Thank you for cleaning this up! :clap: :clap: :clap: 

Actually, while you are in this file, would you mind running [markdownlint](https://github.com/DavidAnson/markdownlint) on the `README`? I use the VSCode IDE and the markdownlint extension works well.

At a minimum, could you also add `yaml` in your contributed fenced code blocks? See review.

File: README.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ None.
 
 Enable or disable the yum-cron service:
 
-    yum_cron_service_enabled: True
+```
```
Comment (giovtorres) [inline README.md]:
```suggestion
```yaml
```

File: README.md [hunk 2]
```diff
@@ -28,130 +30,182 @@ What kind of update to use.  Options are:
 - `security-severity:Critical`         = yum --sec-severity=Critical upgrade
 - `minimal`                            = yum --bugfix update-minimal
 - `minimal-security`                   = yum --security update-minimal
-- `minimal-security-severity:Critical` =  --sec-severity=Critical update-minimal
+- `minimal-security-severity:Critical` = yum --sec-severity=Critical update-minimal
```
Comment (giovtorres) [inline README.md:33]: :+1:

File: README.md [hunk 3]
```diff
@@ -28,130 +30,182 @@ What kind of update to use.  Options are:
...
 
 Use this to filter Yum core messages:
 
 - `-4`: critical
 - `-3`: critical+errors
 - `-2`: critical+errors+warnings (default)
 
-    yum_cron_debuglevel: "-2"
+```
+yum_cron_debuglevel: "-2"
```
Comment (giovtorres) [inline README.md:114]: :clap:

**Comment by giovtorres** [comment]
The Travis build is definitely not failing because of this PR. I think it has to do with Travis changing the default version of Python from 2 to 3 since this build job last ran.

**Comment by kpankonen** [comment]
> Thank you for cleaning this up!   
> 
> Actually, while you are in this file, would you mind running [markdownlint](https://github.com/DavidAnson/markdownlint) on the `README`? I use the VSCode IDE and the markdownlint extension works well.
> 
> At a minimum, could you also add `yaml` in your contributed fenced code blocks? See review.

done!

markdownlint doesn't have any suggestions (it was complaining before this change about mixing the fenced and indented code block styles)

**Review by giovtorres — APPROVED** [review]
LGTM :100: Thank you for addressing feedback!

**Pull request merged**